config/static_data.yml

```diff
@@ -23,10 +23,6 @@ categories:
   - name: Immigration, visas and citizenship
     summary: >
       Australian citizenship, customs, visas, migration and tourism.
-  - name: Infrastructure and telecommunications
-    summary: >
-      Transport safety, television reception, telecommunications.
-    path: /categories/infrastructure-and-telecommunications
```

Comment:
Do we still use any of the categories in this file or are they now coming from the database? If not, maybe we should delete them all and update `config/initializers/static_data.rb` to not load them in?